homeassistant/components/media_extractor/__init__.py

```diff
@@ -147,8 +147,8 @@ def call_media_player_service(self, stream_selector, entity_id):
         if entity_id:
             data[ATTR_ENTITY_ID] = entity_id
 
-        self.hass.async_create_task(
-            self.hass.services.async_call(MEDIA_PLAYER_DOMAIN, SERVICE_PLAY_MEDIA, data)
+        self.hass.create_task(
+            self.hass.services.call(MEDIA_PLAYER_DOMAIN, SERVICE_PLAY_MEDIA, data)
```

Comment:
`create_task` takes a coroutine object as argument.
```suggestion
            self.hass.services.async_call(MEDIA_PLAYER_DOMAIN, SERVICE_PLAY_MEDIA, data)
```